loky/backend/semaphore_tracker.py

```diff
@@ -73,11 +72,13 @@ def ensure_running(self):
                 # process will out live us, so no need to wait on pid
                 exe = spawn.get_executable()
                 args = [exe] + util._args_from_interpreter_flags()
-                if len(args[-1]) > 100:  # pragma: no cover
-                    warnings.warn(
-                        "args_from_flag returned -R+ -> why??", RuntimeWarning)
```

Comment:
Remove this warning. This is caused by a bug in python3.3 for `multiprocessing.util._args_from_interpreter_flags`.
Note that it might be the cause of the failures in `test_process` as noted in #56